frontend/yarn.lock

```diff
@@ -3073,9 +3073,9 @@
     "@types/d3-selection" "^1"
 
 "@types/d3@^3":
-  version "3.5.44"
-  resolved "https://registry.yarnpkg.com/@types/d3/-/d3-3.5.44.tgz#28635f8bb9adf1cef5bbef2b06778174a9d34383"
-  integrity sha512-hFEcf03YGJ2uQoDYEp3nFD5mXWxly5kf6KOstuOQFEs9sUCN7kNlKhcYkpZ3gK6PiHz4XRLkoHa80NVCJNeLBw==
+  version "3.5.47"
```

Comment:
Why is the `yarn.lock` file updated here? The PR does not contain any changes to our dependencies or? But I would propose to directly update our plotly dependency to the most recent version in this PR

Reply:
Isn't the plotly.js dependency upgraded?

Reply:
hmm, yep probably just in our yarn.lock. But I think it might be better to actually update it in the package.json as well